configure.ac

```diff
@@ -4418,13 +4418,6 @@ PAC_C_MACRO_VA_ARGS
 
 # Check for alloca function.  May set HAVE_ALLOCA_H and HAVE_ALLOCA
 AC_FUNC_ALLOCA
```

Comment:
These lines can probably go along with the rest.

Reply:
Yeah, I was pondering on this. I think `alloca` still can be considered on local functions where the allocation is known to be small, then we will need the configure macros to know whether it can be used. 

But I guess I am leaning toward your side. It's easy enough to add it back when we need to.